docs/querying/sql-functions.md

```diff
@@ -498,19 +501,64 @@ Returns the first non-null value.
 
 ## CONCAT
 
-`CONCAT(expr, expr...)`
+Concatenates a list of expressions.
 
-**Function type:** [Scalar, string](sql-scalar.md#string-functions)
+* **Syntax:** `CONCAT(expr[, expr,...])`
+* **Function type:** Scalar, string
 
-Concatenates a list of expressions.
+<details><summary>Example</summary>
+
+The following example concatenates the `OriginCityName` column, the word ` to `, and the `DestCityName` column. The `OriginCityName` and `DestCityName` are both from the `flight-carriers` datasource. 
```

Comment:
```suggestion
The following example concatenates the `OriginCityName` column, the string ` to `, and the `DestCityName` column from from the `flight-carriers` datasource. 
```
Great example. I don't think we need a two-sentence description.

Reply:
changed sentence, but implememted the spirit behind the suggestion